packages/block-editor/src/components/inserter-list-item/style.scss

```diff
@@ -31,8 +31,13 @@
 
 	&:not(:disabled) {
 		&:hover {
+			background: var(--wp-admin-theme-color);
```

Comment:
Nice change:
<img width="890" alt="Screenshot 2021-10-05 at 09 30 40" src="https://user-images.githubusercontent.com/1204802/135979366-87629375-0941-4264-b1f8-75023c5c8598.png">

I will spin up a separate PR to fix the extra top margin present here, and unify the inner paddings of the container, which are a bit off, not as fault of this PR, but just something I noticed.

Reply:
Followup in https://github.com/WordPress/gutenberg/pull/35339.

Reply:
I think we should split the design updates to the inserter content to a different PR. It also adds some delay in the main inserter.